apps/taxes/src/modules/avatax/ui/edit-avatax-configuration.tsx

```diff
@@ -4,13 +4,16 @@ import { useRouter } from "next/router";
 import React from "react";
 import { z } from "zod";
 import { trpcClient } from "../../trpc/trpc-client";
-import { AvataxConnectionObfuscator } from "../avatax-connection-obfuscator";
-import { AvataxConfig } from "../avatax-connection-schema";
+import { AvataxObfuscator } from "../avatax-obfuscator";
+import { AvataxConfig, BaseAvataxConfig } from "../avatax-connection-schema";
 import { AvataxConfigurationForm } from "./avatax-configuration-form";
+import { useAvataxConfigurationStatus } from "./configuration-status";
 
-const avataxObfuscator = new AvataxConnectionObfuscator();
+const avataxObfuscator = new AvataxObfuscator();
 
 export const EditAvataxConfiguration = () => {
+  const [_, setStatus] = useAvataxConfigurationStatus();
```

Comment:
I dont see a reason to use tuple syntax. useState is using it because you *want* to set a semantic name. Here the name should be given by the hook